test/Microsoft.IdentityModel.TestUtils/Default.cs

```diff
@@ -8,6 +8,7 @@
 using System.Security.Claims;
 using System.Security.Cryptography;
 using System.Security.Cryptography.X509Certificates;
+using System.Xml.Linq;
```

Comment:
Why is this added?
```suggestion
```

Reply:
Not needed. they are automatically added by the IDE when writing code